libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -28,7 +30,7 @@ public class IpvSessionItem implements DynamodbItem {
     private AccessTokenMetadata accessTokenMetadata;
     private String errorCode;
     private String errorDescription;
-    private String vot;
+    private @DynamoDBTypeConvertedEnum Vot vot;
```

Comment:
I know I told you to use it, but do we actually need this annotation?

I've noticed that `journeyType` is also enum-valued and seems to 'just work', so maybe it's not needed!

Reply:
I did sort of wonder - while playing around looking at how the enum was serialized. I'll try taking it out and see what happens.

Reply:
Yep, it's happy. Don't need the convertor.